packages/riipen-ui/src/components/Tab.jsx

```diff
@@ -84,121 +43,180 @@ class Tab extends React.Component {
-      orientation,
-      classes
-    );
-
-    return (
-      <React.Fragment>
-        <div
-          tabIndex="0"
-          role="button"
-          aria-pressed={active}
-          className={className}
-          onClick={this.handleChange}
-          onKeyDown={this.handleChange}
-        >
-          {Icon && <Icon className={clsx("icon")} />}
-          {label && <div className={clsx("label")}>{label}</div>}
-        </div>
-        <style jsx>{`
-          .root {
-            align-items: center;
-            border-bottom: 3px solid transparent;
-            cursor: pointer;
-            display: inline-flex;
-            justify-content: center;
-            height: 100%;
-            min-width: 72px;
-            padding: ${theme.spacing(3)}px ${theme.spacing(6)}px;
-            text-align: center;
-            transition: all ${theme.transitions.duration.standard}ms;
-          }
-
-          .root:focus {
-            color: ${theme.palette[color].main};
-          }
-
-          .icon + * {
-            margin-left: ${theme.spacing(2)}px;
-          }
-
-          .icon {
-            margin-bottom: 1px;
-          }
-
-          .label {
-            font-family: ${theme.typography.fontFamily};
-            font-size: 14px;
-            letter-spacing: 2px;
-            line-height: 1.4;
-            text-decoration: none;
-            text-transform: uppercase;
-            width: 100%;
-          }
-
-          .vertical {
-            border-bottom: 0;
-            border-right: 3px solid transparent;
-            padding: ${theme.spacing()}px ${theme.spacing(5)}px;
-            text-align: left;
-          }
-
-          .vertical-active {
-            background-color: ${theme.palette.grey[50]};
-          }
-
-          .primary-active {
-            border-color: ${theme.palette.primary.main};
-            color: ${theme.palette.primary.main};
-          }
-          .primary:hover {
-            color: ${theme.palette.primary.main};
-          }
-          .secondary-active {
-            border-color: ${theme.palette.secondary.main};
-            color: ${theme.palette.secondary.main};
-          }
-          .secondary:hover {
-            color: ${theme.palette.secondary.main};
-          }
-
-          .disabled {
-            opacity: 0.5;
-            pointer-events: none;
-          }
-
-          .full-width {
-            flex-basis: 0;
-            flex-grow: 1;
-            flex-shrink: 1;
-            max-width: none;
-          }
-        `}</style>
-      </React.Fragment>
-    );
-  }
-}
-
-export default withClasses(Tab);
+  const className = clsx(
+    "root",
+    active ? `${color}-active` : null,
+    active ? `${orientation}-active` : null,
+    color,
+    disabled ? "disabled" : null,
+    focusVisible ? "focusVisible" : null,
+    fullWidth ? "full-width" : null,
+    orientation,
+    classes
+  );
+
+  return (
+    <React.Fragment>
+      <div
+        tabIndex="0"
```

Comment:
Should also have `tabIndex=-1` if `disabled` is `true`

I think all the adding `.disabled.focusVisible{ outline:none}` styles in these components + having `tabIndex` mimic `pointer-events` behaviour for `disabled` components should be a separate ticket.